src/utils/arrayUtils.ts

```diff
@@ -25,3 +25,17 @@ export async function callChunked<Item>(
     await processFn(arrayChunk)
   }
 }
+
+/**
+ * Return a copy of the given array without null or undefined values
+ */
+export function removeNullish<T>(array: readonly (T | null | undefined)[]): T[] {
+  return array.filter((e) => e !== undefined && e !== null) as T[]
+}
+
+/**
+ * Return a copy of the given array without falsy values (eg: false, 0, '', null, undefined)
+ */
+export function removeFalsy<T>(array: readonly (T | null | undefined)[]): T[] {
+  return array.filter((e) => e) as T[]
+}
```

Comment:
Finally, I added both because I can think of use cases where both can be useful, but feel free to disagree

Reply:
I think that's a great call!